script/release/08a_communityPushTags.sh

```diff
@@ -3,8 +3,8 @@
 # pushes the tag to github for community releases
 
 # create a tag
-commitMsg="Tagging $tagName"
-./droolsjbpm-build-bootstrap/script/git-all.sh tag -a $tagName -m "$commitMsg"
+commitMsg="Tagging $kieVersion"
+./droolsjbpm-build-bootstrap/script/git-all.sh tag -a $kieVersion -m "$commitMsg"
 
 # pushes tag to github ¡/kiegroup
-./droolsjbpm-build-bootstrap/script/git-all.sh push origin $tagName
\ No newline at end of file
+./droolsjbpm-build-bootstrap/script/git-all.sh push -n origin $kieVersion
```

Comment:
Is `-n` intentionally added? that means `dry-run` execution

Reply:
nop - I have to remove this - good spot (this was done for testing)

Reply:
fixed!